### PR TITLE
F5 Firewall v2

### PR DESCRIPTION
**Pull request by GillSamia**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/20239

## Description
New integration to f5. This integration allows user to manage f5 firewall.
Existing endpoint: 
1.	Policy management: list, apply, export and delete
2.	Policy method – CRUD
3.	File types – CRUD
4.	Cookies – CRUD
5.	Hostname – CRUD
6.	Urls – CRUD
7.	Whitelist Ips – CRUD
8.	Blocking-settings – list and update
9.	GWT profile – CRUD
10.	Json profiles – CRUD
11.	Xml profiles – CRUD
12.	Parameters – CRUD
13.	Signatures – list 
14.	Update enforcer command
15.	Server technologies -list, add and delete.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **Qmasters**

### CODE REVIEW

File: Packs/F5/Integrations/f5_v2/f5_v2.yml [hunk 1]
```diff
@@ -0,0 +1,1719 @@
+category: IT Services
+commonfields:
+  id: F5_firewall
```
Comment (barchen1) [inline Packs/F5/Integrations/f5_v2/f5_v2.yml:3]: please change the id to F5 WAF

File: Packs/F5/Integrations/f5_v2/f5_v2.yml [hunk 2]
```diff
@@ -0,0 +1,1719 @@
...
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Manages F5 firewall
+display: f5 firewall - v2
```
Comment (barchen1) [inline Packs/F5/Integrations/f5_v2/f5_v2.yml:31]: The display name should be F5 Application Security Manager (WAF)

File: Packs/F5/Integrations/f5_v2/f5_v2.yml [hunk 3]
```diff
@@ -0,0 +1,1719 @@
...
+name: F5_firewall
```
Comment (barchen1) [inline Packs/F5/Integrations/f5_v2/f5_v2.yml:32]: The name should be F5 WAF

File: Packs/F5/Integrations/f5_v2/f5_v2.py [hunk 4]
```diff
@@ -0,0 +1,1059 @@
...
+
+            Returns:
+                str: MD5 hash of the policy (can also be called the policy ID).
+        """
+        response = self._http_request(method='GET', url_suffix='asm/policies',
+                                      headers=self.headers,
+                                      params={'items': [{'name': policy_name}]})
+        index = -1
+        for element in response.get('items'):
+            if element.get('name') == policy_name:
```
[the remark this answers is not in the document]
Comment (GillSamia) [reply Packs/F5/Integrations/f5_v2/f5_v2.py:58]: The response is not filtered. I removed the params from the request as it has no effect.

File: Packs/F5/Integrations/f5_ASM/f5_ASM.yml [hunk 5]
```diff
@@ -0,0 +1,3354 @@
+category: IT Services
+commonfields:
+  id: F5 WAF
+  version: -1
+configuration:
+- display: Server URL (e.g. 8.8.8.8)
```
Comment (anara123) [inline Packs/F5/Integrations/f5_ASM/f5_ASM.yml:6]: you ask for server url but give an IP as example/
Change to URL example

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks again for your contribution!
Please add the changes for the update/delete commands as we discussed.
Let's talk over Slack and schedule a demo.
Let me know if you need help.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**